app/src/main/java/com/example/teamyuml/decentworkmobile/views/NoticeList.java

```diff
@@ -147,6 +130,35 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                 fragmentTransaction.commit();
                 drawerLayout.closeDrawers();
                 break;
+            case R.id.noticeList:
```

Comment:
You're repeating this code two times. Make method for it.

Reply:
imo now it's overhead